assets/javascripts/modules/analytics/omniture.js

```diff
@@ -51,6 +53,18 @@ define([
         s.eVar19 = 'D=c19';
         s.prop42 = 'Subscriber';
         s.prop47 = 'UK';
+
+        var main = $('main')[0];
```

Comment:
Hmm, [map and forEach have the same browser support afaik](http://kangax.github.io/compat-table/es5/). Just a reminder we only load the JS app in ES5 (~IE9 environments) so for application JS you can mostly use e5 methods.

Rest sounds good though.